css/media.css

```diff
@@ -0,0 +1,321 @@
+@media(min-width: 320px) {
+    body {
+        box-sizing: border-box;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)